plugins/lsp/language-servers/default.nix

```diff
@@ -508,7 +505,7 @@ let
     {
       name = "ruff-lsp";
       description = "ruff-lsp, for Python";
-      package = pkgs.python3Packages.ruff-lsp;
+      package = pkgs.ruff-lsp or pkgs.python3Packages.ruff-lsp;
```

Comment:
I would have a `TODO` comment on this one.

Reply:
Added a TODO similar to the gdtoolkit rename.